INSTALL

```diff
@@ -243,7 +243,7 @@ ImageMagick
 Optional dependencies
 ^^^^^^^^^^^^^^^^^^^^^
 
-`Pillow <http://python-imaging.github.io/>`__
+`Pillow <https://github.com/python-pillow/Pillow>`__
     If Pillow is installed, matplotlib can read and write a larger
```

Comment:
Shouldn't this be http://python-pillow.github.io/ which looks like the homepage rather than the git repo?